.github/workflows/rc.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           token: ${{ secrets.BOT_GITHUB_TOKEN }}
           git-username: ${{ secrets.BOT_GITHUB_USERNAME }}
 
-      - uses: Activiti/activiti-scripts/.github/actions/wait-for-chart@develop
+      - uses: Activiti/activiti-scripts/.github/actions/wait-for-chart@000995bdf3eae49f78ff39c462226208039ef1a8 # 8.1.0
```

Comment:
Good catch! It's risky referencing `develop`.

Reply:
I saw that we don't update it often so I used the last release of Activiti. I think we will have to update this manually if we will do some change in the action and we want to update the workflow before the next release